packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -0,0 +1,162 @@
+import djPortrait from 'assets/img/DJportrait.jpg'
+import BackgroundWaves from 'components/background-animations/BackgroundWaves'
+import { useMedia } from 'hooks/useMedia'
+import { SignInPage } from 'pages/sign-in-page/SignInPage'
+import { AudiusValues } from 'pages/sign-on-page/AudiusValues'
+import SignUpPage from 'pages/sign-up-page'
+import { SIGN_IN_PAGE, SIGN_UP_PAGE, TRENDING_PAGE } from 'utils/route'
+
+const messages = {
+  newToAudius: 'New to Audius?',
+  createAccount: 'Create an Account'
+}
+
+export const SignOnPage = () => {
+  const { isMobile } = useMedia()
+  const [isExpanded] = useToggle(false)
+  const [isLoaded, setIsLoaded] = useToggle(false)
+  const { spacing } = useTheme()
+
+  useEffect(() => {
+    setIsLoaded(true)
+  }, [setIsLoaded])
+
+  const routes = (
+    <Switch>
+      <Route path={SIGN_IN_PAGE}>
+        <SignInPage />
+      </Route>
+      <Route path={SIGN_UP_PAGE}>
+        <SignUpPage />
+      </Route>
+    </Switch>
+  )
+
+  if (isMobile) {
+    return (
+      <Flex direction='column' w='100%'>
+        <Flex
+          h={isExpanded ? '100%' : '520px'}
+          borderBottomLeftRadius={!isExpanded ? '2xl' : undefined}
+          borderBottomRightRadius={!isExpanded ? '2xl' : undefined}
+          css={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            backgroundColor: 'white',
+            zIndex: 1,
+            transition: '0.5s ease-in-out 0.5s',
+            transform: isLoaded ? 'translateY(0px)' : 'translateY(-100%)'
+          }}
+        >
+          <Link
+            to={TRENDING_PAGE}
+            css={{ position: 'absolute', left: spacing.l, top: spacing.l }}
+          >
+            <IconCloseAlt color='default' />
+          </Link>
+          {routes}
+        </Flex>
+        <Flex
+          direction='column'
+          alignItems='center'
+          pv='2xl'
+          css={{
+            flexGrow: 1,
+            backgroundImage: `radial-gradient(77.16% 77.16% at 50% 51.81%, rgba(91, 35, 225, 0.80) 0%, rgba(113, 41, 230, 0.64) 67.96%, rgba(162, 47, 235, 0.50) 100%), url(${djPortrait})`,
+            backgroundColor: 'lightgray',
+            backgroundSize: 'cover',
+            backgroundPosition: 'bottom'
+          }}
+        >
+          <Box h={520} />
+          <Switch>
+            <Route path={SIGN_UP_PAGE}>
+              <AudiusValues css={{ margin: 'auto' }} />
+            </Route>
+            <Route path={SIGN_IN_PAGE}>
+              <Text
+                variant='title'
+                strength='weak'
+                color='staticWhite'
+                css={{
+                  marginTop: 'auto',
+                  opacity: isLoaded ? 1 : 0,
+                  transition: 'opacity 0.5s ease-in-out 1s'
+                }}
+              >
+                {messages.newToAudius}{' '}
+                <TextLink variant='inverted' _isHovered asChild>
```

Comment:
feels wrong to pass the private `_isHovered`

Reply:
yeah super wrong agreed! but this is an interesting case where the designs have the underline there for mobile. I think this maybe makes sense cause otherwise you wouldn't know what to click. So maybe we make a non _internal prop for this?